File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        alpha = {
            i : chr(ord('A') + i) for i in range(26)
        }
        answer = []
        
        while columnNumber > 0:
            columnNumber -= 1
            answer.append(alpha[columnNumber % 26])
            columnNumber //= 26
    

        return "".join(answer[::-1])

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.rs

impl Solution {
    pub fn convert_to_title(column_number: i32) -> String {
        let mut column_number = column_number;
        let mut answer: Vec<u8> = vec![];

        while column_number > 0 {
            column_number -= 1;
            answer.push(65 + (column_number % 26) as u8);
            column_number /= 26;
        }
        
        answer.reverse();
        return String::from_utf8(answer).unwrap();
    }
}